enigma-core/app/src/networking/ipc_listener.rs

```diff
@@ -268,9 +268,10 @@ pub(self) mod handling {
         }
         let results = db.insert_tuples(&tuples);
         let mut errors = Vec::with_capacity(tuples.len());
-
+        let mut overall_status = 0;
         for ((deltakey, _), res) in tuples.into_iter().zip(results.into_iter()) {
             let status = if res.is_err() {
+                overall_status = 1;
                 FAILED
```

Comment:
This comment  is not related directly to this PR, but this looks bad to have a constant for failure and not to have a constant for success.